scripts/install_toolchain.ps1

```diff
@@ -23,10 +23,6 @@ $templateProjectPath = Get-ChildItem -Path $toolchainPath -Filter "*Draco.Projec
 try { dotnet new uninstall Draco.ProjectTemplates --verbosity quiet } catch { }
 dotnet new install --force $templateProjectPath
 
-# We save the current location and go to the specified path
-Push-Location
-Set-Location $path
-
 # Create a test project in Draco if one doesn't exist yet
 if (!(Get-ChildItem -Path $path -Filter *.dracoproj)) {
     dotnet new console --language draco
```

Comment:
I don't think this is correct now without the path set to the argument

Reply:
It is, you pass the path as an argument for every next command.